posthog/cdp/templates/hog_function_template.py

```diff
@@ -31,6 +31,7 @@ class HogFunctionTemplate:
     description: str
     hog: str
     inputs_schema: list[dict]
+    category: list[str]
```

Comment:
Ah one thing - you need to add this to the api serializer as well otherwise it wont get returned

Reply:
Not entirely sure here

---

So in my local instance I already see the category field returned from this endpoint http://localhost:8000/api/projects/1/hog_function_templates/?format=json
![2024-09-19 at 13 55 55](https://github.com/user-attachments/assets/681bc019-a686-47ae-af0a-be9f396e31c3)

---

When adding `category` to this list [here](https://github.com/PostHog/posthog/blob/dde8ce4c7edec2f58f5eddddb0782f6c659e4b3a/posthog/api/hog_function.py#L81-L100), I am starting to get this error:
```
ImproperlyConfigured: Field name `category` is not valid for model `HogFunction` in `posthog.api.hog_function.HogFunctionSerializer
```

That probably means I'll have to add `category` to the hog function model, but I don't think we want to do that, because we only need the category on the hardcoded templates anyway.

Does that make sense?